config/locales/de.yml

```diff
@@ -689,6 +689,9 @@ de:
       create: "%{model} erstellen"
       submit: "%{model} speichern"
       update: "%{model} aktualisieren"
+    label:
+      contributor:
+        username: Telegram Username
```

Comment:
> If there is nothing speaking against a migration, I would propose migrating username to telegram_username as a follow-up. This would also improve clarity in our own schema and solve this issue.

Yes 👍

Reply:
There already is an issue: #393

Reply:
Alright, will do this as a follow up.